util/buildinfototags.py

```diff
@@ -0,0 +1,44 @@
+import pprint, requests, re, os
+import sys
+from xml.dom import minidom
+from optparse import OptionParser
+import json
+
+pp = pprint.PrettyPrinter(indent=4)
+
+# moar output, set = 1
+debug=0
+
+usage = "Usage: %prog [-n NAME] \n\n\
+This script will convert a buildinfo.json to CSV list of projects, SHAs, and tags/branches."
+parser = OptionParser(usage)
+
+# required
+parser.add_option("-n", dest="name", help="symbolic name to use, jbosstools-4.3.0.Beta1")
+
+(options, args) = parser.parse_args()
+
+if (not options.name):
+    parser.error("Must to specify ALL commandline flags; use -h for help")
+
+j = json.load(sys.stdin) 
```

Comment:
How about...
    # read from standard input, eg., cat buildinfo.json | python buildinfo2tags.py or 
    # curl http://path/to/buildinfo.json | python buildinfo_to_tags.py